app/lib/actions/katello/upstream_subscriptions/bind_entitlements.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Katello
+    module UpstreamSubscriptions
+      class BindEntitlements < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :organization_id, Integer
+          param :pools, Array
+        end
+
+        def plan(organization_id = nil, pools = [])
+          plan_self(organization_id: organization_id, pools: pools)
+          return unless pools.any?
```

Comment:
should this return be above the plan_self?  Any point in calling the run phase if there are no pools?

Reply:
I was thinking that users would be able to see what they did wrong in the task output rather than just not creating the task at all. Would the task still be created if we didn't call `plan_self`?